.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.8.2
+        with:
+          dotnet-version: '3.1.x'
+
+      - name: Build & Test
+        run: make install test
```

Comment:
Doesn't look like any of the tests are being discovered (not just related to this change as I see the same in travis)
![image](https://user-images.githubusercontent.com/6811347/142889545-676f1436-9280-43b8-8c6c-7479ab5f9721.png)

Reply:
I think we'd need a separate ticket to fix all the sendgrid csharp tests. We uncovered that sendgrid-csharp unit tests aren't running either

Reply:
[DI-1844](https://issues.corp.twilio.com/browse/DI-1844) created.